substra/sdk/schemas.py

```diff
@@ -147,14 +147,31 @@ def is_many(self):
         return self.paths and len(self.paths) > 0
 
     @pydantic.model_validator(mode="before")
-    def exclusive_paths(cls, values):  # noqa: N805
+    def exclusive_paths(cls, values: typing.Any) -> typing.Any:  # noqa: N805
```

Comment:
can you remove the`# noqa: N805`?

Reply:
Nop, its because it does not like that the first arg is not "self" :/

Reply:
If this method entends to receive a class as first argument, pydantic recommends decorate with `@classmethod` for a proper type checking ([model_validator](https://docs.pydantic.dev/latest/concepts/validators/#model-validators))

Reply:
nice thx :)